packages/cli/src/commands/setup/auth/auth.js

```diff
@@ -189,7 +204,6 @@ async function getAuthHandler(module) {
  * Check if a user's project's has a module listed as a dependency or devDependency.
  *
  * @param {string} module
- * @returns {boolean}
```

Comment:
How come you removed this?

Reply:
It seemed like it was inferred on hover and it's usually good not to explicitly type returns for that reason?

Reply:
Ahh, yeah, it can infer the type 👍